README.rst

```diff
@@ -14,18 +14,77 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.0.0 or later.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
+
+::
 
-Please see instructions at http://figure.openmicroscopy.org
+    $ pip install omero-figure
 
+Add figure custom app to your installed web apps:
+
+::
+
+    $ bin/omero config append omero.web.apps '"omero_figure"'
+
+Display a link to 'Figure' at the top of the webclient:
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "figure"}]' 
+
+Now restart OMERO.web as normal.
+
+**Warning**:
+
+if OMERO.figure is installed with OMERO version prior to **5.2.6**,
+the url will be https://your-web-server/omero_figure instead of https://your-web-server/figure as previously. This is due to a package re-organization required to distribute the application using a package manager.
+If installed with OMERO **5.2.6 and older**, the url will be back to https://your-web-server/figure.
+
+
+Enabling PDF generation
+-----------------------
+
+This following steps need to perform where the OMERO.server is running.
+
+* Install `reportlab <https://bitbucket.org/rptlab/reportlab>`_ PDF python package:
+
+::
+
+    $ pip install reportlab
+
+**Note**: For python 2.6, you will need Reportlab 2.7
+
+
+* Install script to generate figure. To perform this task, you must be an admin.
+
+The following steps assume that the OMERO.server is already running.
+
+- Go to the directory where ``omero_figure`` has been installed, this could be on a different machine.
```

Comment:
Did you see new approach https://trello.com/c/Sj519BeH/182-add-figure-export-script-to-eel-builds#comment-57ebe380949fa6e2c8cdbf94 ?